### PR TITLE
build(circleci): use codecov orb

### PR DESCRIPTION
**Pull request by jasonschroeder-sfdc**
Remove the deprecated [codecov](https://www.npmjs.com/package/codecov) NPM package, replace with CircleCI orb from https://circleci.com/developer/orbs/orb/codecov/codecov

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ version: 2.1
 orbs:
     node: circleci/node@5.0.2
     browser-tools: circleci/browser-tools@1.4.0
+    codecov: codecov/codecov@3.2.3
```
Comment (trevor-bliss) [inline .circleci/config.yml:5]: How did you verify this orb version is using the non-deprecated codecov uploader? I'm having a little trouble connecting all the dots here.

Comment (jasonschroeder-sfdc) [reply .circleci/config.yml:5]: I removed the deprecated npm package (and npm task `codecov`), replaced it with the `codecov/upload` step. Have a look at the CircleCI report for this PR, too.

**Review by trevor-bliss — APPROVED** [review]

**Review by callenmas14 — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (mohanraj-r) [reply .circleci/config.yml:5]: Is the "codecov" npm dependency still needed in pkg json with this change?

Comment (jasonschroeder-sfdc) [reply .circleci/config.yml:5]: No, it's not needed anymore.

**Pull request merged**